Cognite.Simulator.Utils/ConnectorBase.cs

```diff
@@ -284,6 +291,33 @@ await UpdateIntegrationRows(false, token)
                     .ConfigureAwait(false);
             }
         }
+        
+        /// <summary>
+        /// Task that runs in a loop, checking for new config in extraction pipelines
+        /// </summary>
+        public async Task CheckForNewConfig(CancellationToken token)
```

Comment:
```suggestion
        public async Task CheckRemoteConfig(CancellationToken token)
```
to make it clear it's remote